CRITICAL - 2021-06-14 15:03:24 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:20 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:26 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:32 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:34 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:37 --> The framework needs the following extension(s) installed and loaded: intl.
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(228): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(180): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#3 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#4 {main}
CRITICAL - 2021-06-14 15:04:56 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
CRITICAL - 2021-06-14 15:07:40 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
CRITICAL - 2021-06-14 15:07:41 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
CRITICAL - 2021-06-14 15:07:41 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
CRITICAL - 2021-06-14 15:07:41 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
CRITICAL - 2021-06-14 15:14:22 --> Call to undefined function CodeIgniter\locale_set_default()
#0 E:\xampp7\htdocs\ci\system\bootstrap.php(157): CodeIgniter\CodeIgniter->initialize()
#1 E:\xampp7\htdocs\ci\public\index.php(28): require('E:\\xampp7\\htdoc...')
#2 {main}
ERROR - 2021-06-14 08:58:44 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 08:58:44 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:01:09 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:01:09 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:01:34 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:01:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:01:38 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:01:38 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:03:47 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:03:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:05:03 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:05:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(41): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:05:40 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:05:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(44): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2021-06-14 09:06:04 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 09:06:04 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#4 E:\xampp7\htdocs\ci\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#5 E:\xampp7\htdocs\ci\system\BaseModel.php(687): CodeIgniter\Model->insert(Array, false)
#6 E:\xampp7\htdocs\ci\app\Controllers\Home.php(43): CodeIgniter\BaseModel->save(Array)
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\Home->formValidation()
#8 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2021-06-14 09:16:02 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 09:16:11 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 09:16:56 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 09:16:58 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 22:32:42 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 22:33:22 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-06-14 22:33:33 --> Class "App\Controllers\UserModel" not found
#0 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#1 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
ERROR - 2021-06-14 22:38:01 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:38:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:38:23 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:38:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(14): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:38:32 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:38:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:40:52 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:40:52 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:47:32 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:47:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:47:46 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:47:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-06-14 22:48:45 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2021-06-14 22:48:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 E:\xampp7\htdocs\ci\system\Database\BaseConnection.php(625): CodeIgniter\Database\BaseConnection->initialize()
#1 E:\xampp7\htdocs\ci\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 E:\xampp7\htdocs\ci\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#3 E:\xampp7\htdocs\ci\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(13): CodeIgniter\BaseModel->findAll()
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-06-14 22:50:58 --> Undefined array key "name"
#0 E:\xampp7\htdocs\ci\app\Views\list.php(49): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'E:\\xampp7\\htdoc...', 49)
#1 E:\xampp7\htdocs\ci\system\View\View.php(230): include('E:\\xampp7\\htdoc...')
#2 E:\xampp7\htdocs\ci\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 E:\xampp7\htdocs\ci\system\Common.php(1271): CodeIgniter\View\View->render('list', Array, true)
#4 E:\xampp7\htdocs\ci\app\Controllers\User.php(14): view('list', Array)
#5 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->index()
#6 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-06-14 23:49:02 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(112): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:49:56 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:50:06 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:50:32 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:51:20 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:51:30 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:52:28 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:53:02 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:53:36 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:54:07 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:54:11 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->update('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:55:07 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:55:20 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:55:25 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:56:22 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:57:04 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-06-14 23:57:26 -->  route cannot be found while reverse-routing.
#0 E:\xampp7\htdocs\ci\system\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 E:\xampp7\htdocs\ci\app\Controllers\User.php(111): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 E:\xampp7\htdocs\ci\system\CodeIgniter.php(928): App\Controllers\User->userUpdate('1')
#3 E:\xampp7\htdocs\ci\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#4 E:\xampp7\htdocs\ci\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 E:\xampp7\htdocs\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
